pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -657,3 +655,30 @@ func getNodeMAC(node *corev1.Node) (net.HardwareAddr, error) {
 	}
 	return mac, nil
 }
+
+func getNodeDataNetAddr(node *corev1.Node) (net.IP, error) {
+	peerNodeIP, err := getNodeInbandAddress(node)
```

Comment:
You are right, we might use NodeIP for a short while before Node report its in-band IP. You remind me, maybe we should add NodeRoute related configurations after we read Node reports its in-band address if the "inBandInterface" is configured. Otherwise, we might configure a wrong route/openflow for NodeIP. What do you think?

Reply:
Yes, that sounds the right thing to do.